Gemfile.lock

```diff
@@ -156,10 +157,8 @@ GEM
     em-socksify (0.3.1)
       eventmachine (>= 1.0.0.beta.4)
     erubis (2.7.0)
-    eventmachine (1.2.0.1)
-    eventmachine (1.2.0.1-x64-mingw32)
-    eventmachine (1.2.0.1-x86-mingw32)
-    execjs (2.6.0)
```

Comment:
These dependencies were all added recently in 61483b18bcbfaa054113a67f40515c7bf3e892b2 when Blade 0.5.1 was introduced. It turns out Thin (a dependency of Blade) doesn't work with event machine 1.2.0 so I backed away from it in https://github.com/javan/blade/commit/62c8cb0ad70f66852356adbe0513da8e44a55e66. Blade also removed exact dependencies on coffee-script and sprockets to reduce conflicts with apps that have tighter requirements. So if we want specific versions, we should declare them.